### DISCUSSION TITLE
New maintainer?

### DISCUSSION THREAD

**Discussion by micseydel** [SilentVoid13/Templater#702] [Ideas & Feature Requests]
https://www.obsidianroundup.org/2022-07-23/ says
> SilentVoid is looking for a new maintainer for [Templater](https://github.com/SilentVoid13/Templater). Based on things others have said in Discord, I suspect that even if you're not a coder, just helping consolidate and categorize issues helps developers save a lot of time.

There's no obvious way to send a direct message...

I'm a software engineer between gigs. I don't know Javascript but would like to learn it. I'm also pretty new to Obsidian but can't imagine not using this plugin.

**Comment by AB1908** [comment]
Hi, please redirect this to #700.

**Reply by micseydel** [reply]
Hm, https://github.com/SilentVoid13/Templater/discussions?discussions_q=maintainer turns up https://github.com/SilentVoid13/Templater/discussions/570 but not https://github.com/SilentVoid13/Templater/issues/700

**Reply by micseydel** [reply]
Ah, it's because I searched in discussions and the other is an issue.

**Comment by QuantumNovice** [comment]
Hi, can I contribute?

**Reply by AB1908** [reply]
Contribution is open to everyone but I think the new maintainer needs to be reasonably familiar with the codebase and have sufficient spare time to address merges/fixes.
